database/factories/ModelFactory.php

```diff
@@ -22,3 +22,99 @@
         'remember_token' => str_random(10),
     ];
 });
+
+// User with Centre
+$factory->state(App\User::class, 'withRandomCentre', function () {
+
```

Comment:
Don't you need a raw user and then merge the centre _id into that?

Reply:
It appears you chain states() onto a factory(app\user::class) and it passes the base object through to be fiddled.

Reply:
Ah cool.